cmake/aws-checksums/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required (VERSION 3.1)
```

Comment:
why are you replicating https://github.com/awslabs/aws-checksums/blob/main/CMakeLists.txt here?

Reply:
because I had to modify it, and did not want to create a ceph fork of that project as well.

this is the reason that I replicated "awc-c-common" and "aws-c-event-stream"

Reply:
actually, i'd suggest fork it.